src/validators/is.psc.validator.ts

```diff
@@ -0,0 +1,12 @@
+export const isPscFlagValid = (isPsc: string): boolean => {
+
+  if (!isPsc) {
```

Comment:
Do we want to check for false specifically rather than for the more generic falsy? (as that would cover undefined as well)

Reply:
'false' is a string and won't cover undefined. False is a valid value, the accepted isPsc values can be 'true' or 'false'.

Reply:
i think i see your point, the second check should return false if undefined so the first check might be redundant

Reply:
We could keep the first check for the reason you pointed out above and then here just check for true and return true and then false in all other cases, so
if (isPsc) { return true; } return false;

Reply:
Ok. Will investigate if this first check can be removed.

Reply:
> We could keep the first check for the reason you pointed out above and then here just check for true and return true and then false in all other cases, so if (isPsc === "true") { return true; } return false;

We can't just return true when 'isPsc' is 'true' as an 'isPsc' value of 'false' is also valid.

I'm removing that first check though, as it's not needed.